pkgs/top-level/python-packages.nix

```diff
@@ -6681,8 +6698,23 @@ in {
     inherit pythonOlder;
   };
 
+
   pyphen = callPackage ../development/python-modules/pyphen {};
 
+  pypng = buildPythonPackage rec {
```

Comment:
We are moving Python expressions out of `pkgs/top-level/python-packages.nix` into `pkgs/development/python-modules/<module>/default.nix`. 

Please move the expression there, and call it from `pkgs/top-level/python-packages.nix` using `callPackage ../development/python-modules/<package> { };`.